test/ArrayObjectTest.php

```diff
@@ -103,10 +103,22 @@ public function testAsort()
 
     public function testCount()
     {
+        if (version_compare(PHP_VERSION, '7.2', '>=')) {
+            $this->setExpectedException(
+                'PHPUnit_Framework_Error_Warning',
+                'Parameter must be an array or an object that implements Countable'
+            );
+        }
         $ar = new ArrayObject(new TestAsset\ArrayObjectObjectVars());
         $this->assertEquals(1, $ar->count());
     }
 
+    public function testCountable()
+    {
+        $ar = new ArrayObject(new TestAsset\ArrayObjectObjectCount());
+        $this->assertEquals(42, $ar->count());
```

Comment:
You can use [`assertCount`](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertCount).

Reply:
this is a copy/paste from previous test, I don't want to do refactoring in a PR only design to fix a minor compatibility issue.

Feel free to open a separate PR if you really think this worth the work ;)

Reply:
You add these new lines, so please use the correct method. Thanks!

Reply:
ok.
No time for Bikeshedding.